src/Layer/interface/Core/PaintNode.h

```diff
@@ -41,13 +40,14 @@ class VGG_EXPORTS PaintNode : public Node
   Mask outlineMask;
   EMaskType maskType{ MT_None };
   EBoolOp m_clipOperator{ BO_None };
+  EOverflow m_overflow{ OF_Hidden };
```

Comment:
inconsistent naming style. All private filed names should be prefixed with `m_`, right?